template/.github/workflows/{% if AddLinkCheck %}link-check.yml{% endif %}

```diff
@@ -0,0 +1,38 @@
+name: link-check
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+  schedule:
+  - cron: '0 0 1 * *' # first day of every month at midnight
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1
+
+      - name: Set Issue Title
+        id: set_title
+        run: echo "ISSUE_TITLE=Link Checker Report - $(date '+%Y-%m-%d')" >> $GITHUB_ENV
+
+      - name: Create Issue From File
+        if: env.lychee_exit_code != 0
```

Comment:
The issue should only be made when workflow is triggered by a schedule. When it is not triggered by schedule it should fail the workflow.